### ISSUE TITLE
Output of whisper online bank is different from official whisper log_mel_spectrogram

### ISSUE THREAD

**Issue by jie-chen** [csukuangfj/kaldi-native-fbank#65]
Hi, as the unit test shown below, the two outputs mismatch. Upon a further look, the problem could lie in the different underlying stft approaches.
```
import kaldi_native_fbank as knf
import torch
import whisper

torch.manual_seed(20220825)
sampling_rate = 16000
samples = torch.randn(16000 * 10)
opts = knf.WhisperFeatureOptions()

opts.dim = 80
online_whisper_fbank = knf.OnlineWhisperFbank(opts)

online_whisper_fbank.accept_waveform(sampling_rate=sampling_rate, waveform=samples.numpy())
online_whisper_fbank.input_finished()

features = []
for i in range(online_whisper_fbank.num_frames_ready):
    f = online_whisper_fbank.get_frame(i)
    f = torch.from_numpy(f)
    features.append(f)

features = torch.stack(features).T
log_spec = torch.clamp(features, min=1e-10).log10()
log_spec = torch.maximum(log_spec, log_spec.max() - 8.0)
mel = (log_spec + 4.0) / 4.0
whisper_mel = whisper.log_mel_spectrogram(samples)
for i in range(online_whisper_fbank.num_frames_ready):
    f1 = mel[:, i]
    f2 = whisper_mel[:, i]
    assert torch.allclose(f1, f2, atol=1e-3), (i, (f1 - f2).abs().max())

**Comment by csukuangfj** [comment]
See also https://github.com/csukuangfj/kaldifeat/issues/116
